rcl_yaml_param_parser/test/special_float2.yaml

```diff
@@ -0,0 +1,8 @@
+# config/test_yaml
+---
+
+lidar_ns:
+  lidar_1:
+    ros__parameters:
+      test_nan: [1.1, 2.2, nananana]
```

Comment:
I think I'm really looking for `.nananan`, which should be parsed as a string, rather than a double.

Reply:
That's my misreading, i will change it.